pkgs/by-name/xe/xevd/package.nix

```
@@ -0,0 +1,48 @@
+{
+  lib,
+  fetchFromGitHub,
+  stdenv,
+  unstableGitUpdater,
+  cmake,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "xevd";
+  version = "0.4.1-unstable-2024-03-18";
```

AFAICT, this version has nothing to do with 0.4.1?

```suggestion
  version = "unstable-2024-03-18";
```

Please also add a comment why the unstable version is used rather than a stable tag.

---

Using the stable version now and applying patches needed for it to compile instead.